MFE/devices/src/view/createDevicesCSV/steps/SummaryStep/SummaryStep.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from 'react';
+import { Box, Button, CircularProgress, Typography } from '@material-ui/core';
+import { Done } from '@material-ui/icons';
+import PropTypes from 'prop-types';
+import { Trans, useTranslation } from 'react-i18next';
+import { useSummaryStepStyles } from './style';
+import DevicesErrorTable from './DevicesErrorTable';
+
+const SummaryStep = ({
+  handleCreateDevicesCSV,
+  createdDevices,
+  notCreatedDevices,
+  isLoadingCreateDevicesCSV,
+  handleGoToDevicesPage,
+}) => {
+  const classes = useSummaryStepStyles();
+  const { t } = useTranslation('createDevicesCSV');
+
+  useEffect(() => {
```

Comment:
I don't think that this is the best way of calling the `handleCreateDevicesCSV` function. You can call it when the user clicks the button to go to the summary step.